### DISCUSSION TITLE
How to commit my plugin

### DISCUSSION THREAD

**Discussion by SkyWave2022** [PlaceholderAPI/PlaceholderAPI#839] [(2) Support] [answered]
myplugin supports placeholderapi and want to commit so players can use ecloud to download my expansion. How to commit my plugin
![2022-05-25_13 46 12 (1)](https://user-images.githubusercontent.com/86274592/172120596-24a8f127-cf83-4d0f-8082-bf8f6a83d68b.png)
>

**Comment by Andre601** [comment, marked as answer]
Not sure what you mean with "commit"...

If you refer to the [wiki](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki), check the [README.md on the `wiki` branch](https://github.com/PlaceholderAPI/PlaceholderAPI/blob/wiki/README.md).
